src/main/java/org/aksw/gerbil/datatypes/ExperimentType.java

```diff
@@ -285,11 +321,22 @@ public boolean equalsOrContainsType(ExperimentType type) {
         }
         case OKE2018Task4:{
         	switch(type) {
+            case OKE2018Task4:
         	case RE:
-        	case OKE2018Task4:
+        	case KE:
```

Comment:
This is not true. OKE2018Task4 contains itself (of cause) but not KE.
Apart from that, the whole A2KB and its "sub tree" is missing, for which this case should return true, as well.